django/core/management/commands/makemigrations.py

```diff
@@ -101,9 +101,12 @@ def handle(self, *app_labels, **options):
         aliases_to_check = connections if settings.DATABASE_ROUTERS else [DEFAULT_DB_ALIAS]
         for alias in sorted(aliases_to_check):
             connection = connections[alias]
-            if (connection.settings_dict['ENGINE'] != 'django.db.backends.dummy' and
-                    # At least one app must be migrated to the database.
-                    any(router.allow_migrate(connection.alias, label) for label in consistency_check_labels)):
+            if (connection.settings_dict['ENGINE'] != 'django.db.backends.dummy' and any(
+                    # At least one model must be migrated to the database.
+                    router.allow_migrate(connection.alias, label, model_name=model.__name__)
```

Comment:
They should always be the same but you might want to use `model._meta.object_name` instead.